README.md

```diff
@@ -14,7 +14,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://Andrew1256.github.io/layout_landing-page/)
```

Comment:
Add demo link to the PR description 
<img width="946" alt="image" src="https://github.com/user-attachments/assets/91a2088a-0545-40bd-9ab5-1bf39adf5012" />

Reply:
https://andrew1256.github.io/layout_landing-page/